### ISSUE TITLE
Not Working on Linux Mint 20

### ISSUE THREAD

**Issue by gbkelley** [EpicCash/epic-gui-wallet#27]
EPIC GUI Wallet 3.0.1 installed on Linux Mint 20 does not work. It will run but will not let you create a new wallet, just returns 'unknown error'. Creates the epic_3-0-0_config.json file in /home/user/.epic folder but file only shows "{ account dirs": [] }. Seems like it doesn't know how to set the default folder for the wallet files. It also won't restore using seed phrase (unknown error again) for a CLI created wallet. And it won't allow login to a CLI created wallet (wait wheel just spins forever).

I have attempted to install this on three different systems running Mint 20, all with same results.

The Windoze version installed fine and created a new wallet but Linux is FUBAR.

I also tried to install using the latest Ubuntu v22 LTS live CD with same results - unknown error when creating new wallet.

**Comment by gbkelley** [comment]
Moving on to v4

**Comment by sang-shelton** [comment]
Same issue here

![Screenshot_2024-04-25_18-10-08](https://github.com/EpicCash/epic-gui-wallet/assets/154356073/648c72bd-ad78-4f19-b8a3-71ead1aefeda)

```
./EpicWallet-5.3.0.AppImage 
18:03:27.864 › App starting...
noderuninbackground false
epicboxruninbackground false
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[48149:0425/180328.157257:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
18:03:28.211 › Checking for update
18:03:29.904 › Update for version 5.3.0 is not available (latest version: 5.3.0, downgrade is allowed).
no update available
```
